src/EntraModuleBuilder.ps1

```diff
@@ -229,39 +229,36 @@ Set-StrictMode -Version 5
 
 # Main function to create the root module
 [void] CreateRootModule([string] $Module) {
-    # Determine the root module name based on the module type
-    $rootModuleName = if ($Module -eq 'Entra') {
-        'Microsoft.Entra.psm1'
-    } else {
-        'Microsoft.Entra.Beta.psm1'
-    }
-
-    # Get the list of submodules and exclude the root module
-    $subModuleFiles = $this.GetSubModuleFiles($Module, $this.OutputDirectory)
-    $subModules = @()
-    
-    # Prevents the old root module from being added to avoid cyclic dependencies
-    foreach ($module in $subModuleFiles) {
-        if ($module -ne $rootModuleName) {
-            $subModules += $module
-        }
+    #We only generate the .psm1 file with Enable-EntraAzureADAlias if it's 'Microsoft.Entra' module
```

Comment:
I tested the change in this PR and it works. However I don't understand the change in this function